exec/java-exec/src/main/java/org/apache/drill/exec/server/rest/WebServer.java

```diff
@@ -159,6 +163,14 @@ public void start() throws Exception {
       servletContextHandler.setSessionHandler(createSessionHandler(servletContextHandler.getSecurityHandler()));
     }
 
+    if (config.getBoolean(ExecConstants.HTTP_ENABLE_CORS)) {
+      FilterHolder cors = servletContextHandler.addFilter(CrossOriginFilter.class, "/*", EnumSet.of(DispatcherType.REQUEST));
+      cors.setInitParameter(CrossOriginFilter.ALLOWED_ORIGINS_PARAM, "*");
```

Comment:
Not so much experience regarding CORS, but should we expect admins to need to restrict this to a specific set of origins? I believe so, but I hope people more experienced regarding web security to comment on that. My take is that we should probably make it configurable too. Default to * is probably although not as restrictive as the same-origin policy used when filter is disabled.